integration_test/build-adapters.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# imageProject commitShortSha
+
+base_name="gcr.io/${1}/healthcare-api-dicom-dicomweb-adapter-"
+
+build_adapter () {
+  gradle dockerBuildImage
+  adapter_name=${base_name}${1}
+  version=$(gradle printVersion  | grep -Eo "[0-9]{1,3}\.[0-9]{1,3}" | head -1)
+  postfix=$(gcloud container images list-tags "${adapter_name}" --sort-by=~timestamp | grep " $version" |  grep -Eo "([0-9]{1,3}\.){2}[0-9]{1,3}" | head -1 | grep -Eo "[0-9]{1,3}" | tail -1)
+  postfix=$(($postfix+1))
+  gradle dockerBuildImage -Pdocker_tag=${adapter_name}:$version.$postfix
```

Comment:
One minor thing that came up from internal review, can we add the prefix "v" here? So instead of X.Y.Z, the containers will be labeled as vX.Y.Z. After that I'll merge this PR and configure the post submit build with the publish flag set to true.